docs/components/tuner/callbacks.md

```diff
@@ -32,6 +33,42 @@ tuner = PytorchTuner(..., callbacks=[logger])
 
 You should then be able to see your training runs in wandb.
 
+## Checkpoints
+
+On long train jobs, you may want to periodically save the progress, so you can continue
+from this checkpoint later, if training gets interrupted. Or you may want to save the
+best model, so that you can use it after the training finishes. For these purposes, we
+offer {class}`~finetuner.tuner.callback.training_checkpoint.TrainingCheckpoint` and {class}`~finetuner.tuner.callback.best_model_checkpoint.BestModelCheckpoint`, respectively.
+
+For the  {class}`~finetuner.tuner.callback.training_checkpoint.TrainingCheckpoint` checkpoint, you would simply add it to `callbacks`. Later you could then load the tuner from the checkpoint, as in the example below
+
+
+```python
+from finetuner.tuner.callback import TrainingCheckpoint
+from finetuner.tuner.pytorch import PytorchTuner
+
+checkpoint = TrainingCheckpoint('checkpoints')
+
+tuner = PytorchTuner(..., callbacks=[checkpoint])
+
+# Afterwards, load tuner from the saved theckpoint
+TrainingCheckpoint.load(tuner, 'checkpoints/saved_model_epoch_10')
```

Comment:
is it correct that the `saved_model_epoch_10` is generated automatically?

Reply:
It's what a checkpoint saved in epoch 10 would look like, yes. I can try adding an example for tf/keras as well

Reply:
Actually, looking at this - it would be exactly the same for keras/paddle (just `KerasTuner` instead of `PytorchTuner`), so here I would not add the examples for now